packages/m/mathjax.json

```diff
@@ -20,7 +20,14 @@
       {
         "basePath": "",
         "files": [
-          "es5/**/!(node-main.js)",
+          "**/!(node-main.js)",
```

Comment:
We tend to avoid globstars, and especially open ones -- this line is now basically saying "add every file ever, except node-main.js", which I suspect is not what you want?

Reply:
3.x.x has the `es5` directory, if we do not need to think about it, we can apply this change, thanks for reply.